modules/core/src/main/java/org/apache/ignite/internal/client/impl/GridClientClusterStateImpl.java

```diff
@@ -54,16 +53,6 @@ public GridClientClusterStateImpl(
         super(client, nodes, filter, balancer);
     }
 
-    /** {@inheritDoc} */
-    @Override public void active(final boolean active) throws GridClientException {
-        state(active ? ACTIVE : INACTIVE, true);
-    }
-
-    /** {@inheritDoc} */
-    @Override public boolean active() throws GridClientException {
-        return withReconnectHandling(GridClientConnection::currentState).get();
```

Comment:
`GridClientConnection::currentState` now unused. Seems can be removed too

Reply:
Fixed.